packages/turf-moran-index/index.ts

```diff
@@ -86,11 +86,11 @@ function moranIndex(
   });
 
   const y: number[] = [];
-  featureEach(fc, (feature) => {
+  for (const { feature } of iterFeatures(fc)) {
```

Comment:
This need for destructuring is a little annoying, because we have to also carry featureIndex along. But we're using featureIndex in places, and I'd want signature parity with other iterator tools that will also need to bring along their own other indexes as well.